content/pages/office-policy.md

```diff
@@ -0,0 +1,42 @@
+Title: Office Policy
+Category: pages
+Status: hidden
+Slug: office-policy
+
+(Version 2.0 March 8th 2015)
+
+# Personnel #
+
+* If the office is open for office hours, drop-ins or for any other reason, a MINIMUM of ONE committee member of WiCS must be present in the office at all times.
+* All WiCS members occupying the office are responsible for complying with and enforcing the Code of Conduct.
+
+# Cleanup: #
```

Comment:
I don't like the colons after each section, it looks out of place. I think the header formatting is sufficient. Also not sure that h1 level headings are necessary here, that seems a bit large. Maybe h2?